assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -137,24 +88,16 @@ export const TelemetryOptInBanner = forwardRef(
 
     return visible ? (
       <Banner
-        ref={ref}
         backgroundUrl={`${assetsURL}images/dashboard/analytics-banner-bg.png`}
+        closeButtonLabel={__('Dismiss telemetry', 'web-stories')}
```

Comment:
```suggestion
        closeButtonLabel={__('Dismiss telemetry banner', 'web-stories')}
```

"Dismiss telemetry" sounds weird